test/cache-fs.spec.ts

```diff
@@ -60,5 +71,54 @@ describe(`the cache file system implementation`, () => {
                 .then(() => fs.loadDirectoryTree())
                 .then(() => expect(Date.now() - startTimestamp).to.be.lessThan(timeout * 2))
         })
+
```

Comment:
please don't use slowFs and delay for rescan tests.
The timing is already tested elsewhere, these tests should be much simpler and faster.